docs/more-info/webhooks.mdx

```diff
@@ -1,8 +1,157 @@
+  "ping": "<random-sequence>"
+}
+```
+
+...and respond to this request by sending back the `<random-sequence` as below:
+
+```json
+{
+  "pong": "<random-sequence>"
+}
+```
+
+You can check the [webhook mock server available on our repository](https://github.com/metriport/metriport/blob/master/utils/src/mock-webhook.ts)
+for a simple implementation of this message.
+
+### User data message
+
+These are the messages Metriport will send to your app with user data from Providers, as soon
+as those become available.
+
+The format follows:
+
+```json
+{
+  "meta": {
+    "messageId": "<message-id>",
+    "when": "<date-time-in-utc>"
+  },
+  "users": [
+    {
+      "userId": "<user-id-1>",
+      "sleep": [
+        {
+          "metadata": {
+            "date": "2022-12-21",
+            "source": "garmin"
+          },
+          "start_time": "2022-12-12T22:15:40Z",
+          "end_time": "2022-12-13T04:15:40Z",
+          "durations": {
+            "awake_seconds": 400,
+            ...
+          },
+          "biometrics": {
+            "heart_rate": {
+              "avg_bpm": 70,
+              ...
+            },
+            ...
+          }
+        },
+        {
+          "metadata": {
+            "date": "2022-12-21",
+            "source": "garmin"
+          },
+          "start_time": "2022-12-12T22:15:40Z",
+          "end_time": "2022-12-13T04:15:40Z",
+          "durations": {
+            "awake_seconds": 400,
+            ...
+          },
+          "biometrics": {
+            "heart_rate": {
+              "avg_bpm": 70,
+              ...
+            },
+            ...
+          }
+        }
+      ],
+      "activity": [
+        {
+          "metadata": {
+            "date": "2022-12-21",
+            "source": "garmin"
+          },
+          "summary": {
+            "biometrics": {
+              ...
+            },
+            "durations": {
+              ...
+            },
+            ...
+          },
+          "activity_logs": [
+            ...
+          ],
+        }
+      ]
+    },
+    {
+      "userId": "<user-id-2>",
+      ...
+    }
+  ]
+}
+```
+
+Where:
+
+- `message-id`: this request ID from Metriport, useful for debugging purposes only;
+- `date-time-in-utc`: the timestamp in datetime format when this message was originally sent
+  (example: `2022-12-24T00:46:05.413Z`)
+- `users`: array of users, where each item represents all of that user's information available
+  in this request (more data might be available in subsequent requests)
+- for each user:
+  - `user-id`: the user ID as defined by your application when it called [`/create-user`](/user/create-user)
+  - `data`: array of data points for the given user; this follows the same format of the response
+    of the SDK API reference
+    - in the example above, `user-id-1` has 2 entries of [`sleep`](/health-device-api-reference/get-sleep-data)
+    data and 1 entry of [`activity`](/health-device-api-reference/get-activity-data) data
```

Comment:
let's just use this to describe the response https://mintlify.com/docs/api-components/expandable , will make it cleaner